src/utils.rs

```diff
@@ -33,3 +46,26 @@ pub fn host() -> Result<String, Error> {
 
     Ok(host)
 }
+
+/// Get global `self` and provide way to pass expression to run on it because there is no common abstraction.
```

Comment:
What's an example usecase of this macro?

Reply:
At the moment I used it only to make the fetch service work in workers too, but in the future this can be used to access the cache, indexed db or offscreen canvas API.

Reply:
Ok, can we add this macro to https://github.com/yewstack/yew/pull/867 instead then?

Reply:
Done.